diff --git a/dist/core.esm.js b/dist/core.esm.js
index 1577c253774e3c37eafa4f267f1944426deee405..12722f11f8c40c61bfceadd34961b9f19d53a744 100644
--- a/dist/core.esm.js
+++ b/dist/core.esm.js
@@ -1153,7 +1153,7 @@ class KeyboardSensor {
 
   attach() {
     this.handleStart();
-    this.windowListeners.add(EventName.Resize, this.handleCancel);
+    //this.windowListeners.add(EventName.Resize, this.handleCancel);
     this.windowListeners.add(EventName.VisibilityChange, this.handleCancel);
     setTimeout(() => this.listeners.add(EventName.Keydown, this.handleKeyDown));
   }
@@ -1441,7 +1441,7 @@ class AbstractPointerSensor {
       this.listeners.add(events.cancel.name, this.handleCancel);
     }
 
-    this.windowListeners.add(EventName.Resize, this.handleCancel);
+    //this.windowListeners.add(EventName.Resize, this.handleCancel);
     this.windowListeners.add(EventName.DragStart, preventDefault);
     this.windowListeners.add(EventName.VisibilityChange, this.handleCancel);
     this.windowListeners.add(EventName.ContextMenu, preventDefault);
